app/src/main/java/io/gnosis/safe/utils/OwnerKeyHandler.kt

```diff
@@ -0,0 +1,91 @@
+package io.gnosis.safe.utils
+
+import pm.gnosis.model.Solidity
+import pm.gnosis.svalinn.common.PreferencesManager
+import pm.gnosis.svalinn.common.utils.edit
+import pm.gnosis.svalinn.security.EncryptionManager
+import pm.gnosis.utils.*
+import java.math.BigInteger
+
+interface PrivateKeyHandler {
```

Comment:
do we need this interface?

Reply:
I merged the two Interfaces. will also introduce the owner type. I find an interface helful to discover what I can do with a class even if we're not planning to replace the implementation in the near future.